docs/_snippets/installation/plugins-mapping/plugins-mapping.html

```diff
@@ -0,0 +1,848 @@
+         </tr>
+         <tr>
+            <td>
+               <p style="margin-left:0px;"><a href="https://ckeditor.com/cke4/addon/horizontalrule">horizontalrule</a></p>
+            </td>
+            <td>
+               <p style="margin-left:0px;"><a href="https://ckeditor.com/docs/ckeditor5/latest/features/horizontal-line.html">Horizontal line</a></p>
+            </td>
+         </tr>
+         <tr>
+            <td>
+               <p style="margin-left:0px;"><a href="https://ckeditor.com/cke4/addon/htmlwriter">htmlwriter</a></p>
+            </td>
+            <td>N/A</td>
+         </tr>
+         <tr>
+            <td>
+               <p style="margin-left:0px;"><a href="https://ckeditor.com/cke4/addon/iframe">iframe</a></p>
+            </td>
+            <td>N/A</td>
+         </tr>
+         <tr>
+            <td>
+               <p style="margin-left:0px;"><a href="https://ckeditor.com/cke4/addon/iframedialog">iframedialog</a></p>
+            </td>
+            <td>N/A</td>
+         </tr>
+         <tr>
+            <td>
+               <p style="margin-left:0px;"><a href="https://ckeditor.com/cke4/addon/image">image</a></p>
+            </td>
+            <td>
+               <p style="margin-left:0px;"><a href="https://ckeditor.com/docs/ckeditor5/latest/features/images/images-overview.html">Images</a></p>
+            </td>
+         </tr>
+         <tr>
+            <td>
+               <p style="margin-left:0px;"><a href="https://ckeditor.com/cke4/addon/image2">image2</a></p>
+            </td>
+            <td>
+               <p style="margin-left:0px;"><a href="https://ckeditor.com/docs/ckeditor5/latest/features/images/images-overview.html">Images</a></p>
+            </td>
+         </tr>
+         <tr>
+            <td>
+               <p style="margin-left:0px;"><a href="https://ckeditor.com/cke4/addon/imagebase">imagebase</a></p>
+            </td>
+            <td>
+               <p style="margin-left:0px;"><a href="https://ckeditor.com/docs/ckeditor5/latest/features/images/images-overview.html">Images</a></p>
+            </td>
+         </tr>
+         <tr>
+            <td>
+               <p style="margin-left:0px;"><a href="https://ckeditor.com/cke4/addon/indent">indent</a></p>
+            </td>
+            <td>
+               <p style="margin-left:0px;"><a href="https://ckeditor.com/docs/ckeditor5/latest/features/indent.html#indenting-lists">Indent</a></p>
+            </td>
+         </tr>
+         <tr>
+            <td>
+               <p style="margin-left:0px;"><a href="https://ckeditor.com/cke4/addon/indentblock">indentblock</a></p>
+            </td>
+            <td>
+               <p style="margin-left:0px;"><a href="https://ckeditor.com/docs/ckeditor5/latest/features/indent.html">Indent block</a></p>
+            </td>
+         </tr>
+         <tr>
+            <td>
+               <p style="margin-left:0px;"><a href="https://ckeditor.com/cke4/addon/indentlist">indentlist</a></p>
```

Comment:
I'd say this is brought by [`ListEditing`](https://ckeditor.com/docs/ckeditor5/latest/api/module_list_list_listediting-ListEditing.html) but exposed by `Indent` as a command and UI.